src/ParodusInternal.c

```diff
@@ -131,6 +131,27 @@ char* getWebpaConveyHeader()
     return NULL;
 }
 
+int readFromFile(const char *file_name, char **data)
+{
+	FILE *fp;
+	int ch_count = 0;
+	fp = fopen(file_name, "r+");
+	if (fp == NULL)
+	{
+		ParodusError("Failed to open file %s (errno %d)\n", file_name, errno);
+		return 0;
+	}
+	fseek(fp, 0, SEEK_END);
+	ch_count = ftell(fp);
+	fseek(fp, 0, SEEK_SET);
+	*data = (char *) malloc(sizeof(char) * (ch_count + 1));
+	fread(*data, 1, ch_count,fp);
```

Comment:
@schmidtw We are assuming the close-reason-file will just have the reason in plain text without any format.
Is this fine or should we use json file with key "reason"?

Reply:
Just plain text makes sense.